model.py

```diff
@@ -3208,14 +3224,15 @@ def merge_into(self, other_work):
             for pool in w.license_pools:
                 if not pool.open_access:
                     raise ValueError(
+
                         "Refusing to merge %r into %r because it would put an open-access LicensePool into the same work as a non-open-access LicensePool." %
                         (self, other_work)
                         )
 
         my_pwids = self.pwids
         other_pwids = other_work.pwids
-        if not my_pwids.issubset(other_pwids):
-            difference = my_pwids.difference(other_pwids)
+        if not my_pwids == other_pwids:
+            difference = my_pwids.symmetric_difference(other_pwids)
             raise ValueError(
                 "Refusing to merge %r into %r because it has permanent work IDs not present in the target work: %s" % (
```

Comment:
I was wrong about how symmetric_difference works; I made the error message more general.